teuthology/misc.py

```diff
@@ -550,27 +550,31 @@ def copy_file(from_remote, from_path, to_remote, to_path=None):
     ])
 
 
-def move_file(remote, from_path, to_path, sudo=False):
+def move_file(remote, from_path, to_path, sudo=False, preserve_perms=True):
```

Comment:
So the description says the desire is to make this workable even if the destination doesn't exist, but, the method could simply ignore the 'no perms to update' without requiring the caller to request it.  The issue is, however, what permissions for the destination file ought to be used (whatever the source has, or some default, or?...)

I think I'd be happier if the change here were to add a permission for the case of the file not existing (as an optional kw parameter, still).  If to_path doesn't exist, then use perm if supplied, otherwise default to the perm of from_path.  What do you think?